pagai/engine/main.py

```diff
@@ -73,7 +70,8 @@ def initialise(self, force_retrain=False):
         # Check model isn't already trained
         if pickle_file.is_file() and not force_retrain:
             logging.warning("Engine already initiated. Loading engine...")
-            engine = pickle.load(pickle_file)
+            with open(pickle_file, "rb") as file:
```

Comment:
nit: I think `file` is a reserved keyword in python (like id), could you use something else ? it does not affect the program but it messes with the syntactic coloration

Reply:
same below